libr/bin/p/bin_xtr_fatmach0.c

```diff
@@ -105,14 +112,18 @@ static RBinXtrData * oneshot(RBin *bin, const ut8 *buf, ut64 size, int idx) {
 	RBinXtrData *res = NULL;
 	int narch;
 
-
 	arch = r_bin_fatmach0_extract (fb, idx, &narch);
 	if (!arch) {
 		free_xtr (xtr_obj);
 		return res;
 	}
 
-	res = r_bin_xtrdata_new (xtr_obj, free_xtr, arch->b, arch->offset, arch->size, narch);
+	RBinXtrMetadata *metadata = R_NEW0(RBinXtrMetadata);
+
+	metadata->bits = MACH0_(get_bits)(r_buf_buffer(arch->b));
```

Comment:
Missing spaces beforw (

> On 01 Jul 2016, at 07:36, Aneesh Dogra <notifications@github.com> wrote:
> 
> In libr/bin/p/bin_xtr_fatmach0.c:
> 
> >  	arch = r_bin_fatmach0_extract (fb, idx, &narch);
> >  	if (!arch) {
> >  		free_xtr (xtr_obj);
> >  		return res;
> >  	}
> >  
> > -	res = r_bin_xtrdata_new (xtr_obj, free_xtr, arch->b, arch->offset, arch->size, narch);
> > +	RBinXtrMetadata *metadata = R_NEW0(RBinXtrMetadata);
> > +
> > +	metadata->bits = MACH0_(get_bits)(r_buf_buffer(arch->b));
> same comment as previous.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>